src/frame/vulkan.rs

```diff
@@ -32,91 +32,109 @@ pub struct Vulkan {
 impl Vulkan {
     pub fn new() -> Result<Self, Box<dyn Error>> {
         let app_name = CString::new("wluma")?;
-        let app_info = vk::ApplicationInfo::builder()
```

Comment:
not that it is super important, but just reading the [ash changelog](https://github.com/ash-rs/ash/blob/master/Changelog.md), I think the devs intended you to replace `::builder()` with `::default()` and keep all the other code as is, i.e. in this case:

```rust
        let app_info = vk::ApplicationInfo::default()
            .application_name(&app_name)
            .application_version(WLUMA_VERSION)
            .engine_name(&app_name)
            .engine_version(WLUMA_VERSION)
            .api_version(VULKAN_VERSION);
```